hack/update-codegen-dockerized.sh

```diff
@@ -41,7 +41,7 @@ MOCKGEN_TARGETS=(
   "pkg/agent/cniserver/ipam IPAMDriver testing"
   "pkg/agent/flowexporter/connections ConnTrackDumper,NetFilterConnTrack testing"
   "pkg/agent/interfacestore InterfaceStore testing"
-  "pkg/agent/memberlist Memberlist testing"
+  "pkg/agent/memberlist Interface,Memberlist . mock_memberlist.go"
```

Comment:
I don't get why this change is made. Is it related to my comment here https://github.com/antrea-io/antrea/pull/4611#discussion_r1104505828? If yes, I think it's misunderstood. I meant querier interface should provide a method to return memberlist.Interface, instead of providing a method GetAliveNodes and playing as a proxy.

Reply:
@tnqn 
This change is because :-
1. I needed to mock `Interface` of memberlist package which has `AliveNodes` method in order to write unit test (pkg/agent/apiserver/handlers/memberlist/handler_test.go) [Line 75 and 78].
2. Now, after mocking `Interface` of memberlist package in `testing` sub package i was getting cyclic import issue, so to avoid that I created mock in the memberlist package itself.
3. Now, mock file was created as `mock_memberlist_test.go`, I changed that to `mock_memberlist.go` inorder to make methods exportable as I needed it in unit test (handler_test.go).

Reply:
Could you describe the cyclic import? I don't see it from the imports even if the file is in testing package.

Reply:
If we create mock for interfaces `Memberlist` and `Interface` in a file inside `testing` subpackage, it would import "antrea.io/antrea/pkg/agent/memberlist" as it is required for mock of method `AddClusterEventHandler` of interafce `Interface`.
Now, we get cyclic import error in cluster.go and cluster_test.go as cluster_test.go imports this `testing` subpackage.

Reply:
OK, now I see there is a MockMemberlist in the testing package. Normally the testing has a mock implementation of the interface its parent package provides, and it's intended to be used by the interface's consumers in their tests, and the parent package wouldn't consume its own testing (and the testing package will likely consume the parent package because of the type import).

However, just moving the two mock structs to the parent package would cause other "problems":
If the mock structs are put in a file with "_test.go" suffix, they won't be exported and we can't use MockInterface.
If the mock structs are put in a file without "_test.go" suffix, they will be compiled and linked into the product code unnecessarily.

I would suggest to follow the mode that keeps the mock implementation of the target interface in testing package, and keeps the mock implementation of the dependent interface in "mock_memberlist_test.go" in the parent package as it's not intended to be consumed out of the package.

Reply:
@tnqn done, thanks for the suggestion.